pom.xml

```diff
@@ -243,6 +243,11 @@
           <outputDirectory>${project.build.directory}/javadoc</outputDirectory>
           <reportOutputDirectory>${project.reporting.outputDirectory}/javadoc</reportOutputDirectory>
           <sourcePath>src</sourcePath>
+          <quiet>true</quiet>
+          <failOnError>false</failOnError>
+          <doclint>none</doclint>
```

Comment:
<quiet> control if shutting off non-error and non-warning messages.  It is a non-relevant configuration and should be moved in the change.
<failOnError> Specifies if the build will fail if there are errors during Javadoc execution or not.  It should setup as default(True)
<doclint> Specifies specific checks to be performed on Javadoc. It is a non-relevant configuration and should be moved in the change.
<additionalparam> I did not find an explanation on this, it also should be moved 
<noindex> will omit the index from the generated docs. This will generate javadoc without jquery

Reply:
Hi Yuzhan, can you remove all other settings other than noindex? Also in your commit messages, please mention your fix is to mitigate security vulnerability in jszip and jquery